backend/core/serializers.py

```
@@ -1416,22 +1416,95 @@ class TaskTemplateReadSerializer(BaseModelSerializer):
+    )
+
     class Meta:
         model = TaskTemplate
         fields = "__all__"
 
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+
+        try:
+            task = (
+                TaskNode.objects.filter(task_template=instance)
```

Ruff (0.8.2)

`TaskNode` may be undefined, or defined from star imports

(F405)